app/frontends/jobs.ts

```diff
@@ -37,16 +38,16 @@ function getLinksForDisplay(job: Job, urlRoot: string): JobLink[] {
   let { links } = job;
   const dataLinks = job.getRelatedLinks('data');
   if (containsS3DirectAccessLink(job)) {
-    links.unshift(getCloudAccessJsonLink(urlRoot));
-    links.unshift(getCloudAccessShLink(urlRoot));
+    links.unshift(new JobLink(getCloudAccessJsonLink(urlRoot)));
+    links.unshift(new JobLink(getCloudAccessShLink(urlRoot)));
```

Comment:
I have no problem renaming the table - that wouldn't be bad and I'd definitely rather do that before we merge anything in. But I'm not quite following (it is late in the day for me). Are you suggesting the database table should be called job_results instead of job_links? What else did you want to change?